apps/web/src/pages/templates/components/TemplateEditorFormProvider.tsx

```diff
@@ -183,7 +194,7 @@ const TemplateEditorFormProvider = ({ children }) => {
           },
         });
         setTrigger(response.triggers[0]);
-        reset(payloadToCreate);
+        reset(form);
```

Comment:
reset with the current form state, this removed dirty flags from the fields and "disables" the "update" button